.env.example

```
@@ -25,4 +25,13 @@ GOOGLE_CALLBACK_URL="google_callback_url_here"
 # Cloudinary
 CLOUDINARY_NAME="your_cloudinary_name_here"
 CLOUDINARY_KEY="your_cloudinary_key_here"
-CLOUDINARY_SECRET="your_cloudinary_secret_here"
\ No newline at end of file
+CLOUDINARY_SECRET="your_cloudinary_secret_here"
+
+# Redis
+REDIS_URL="redis://localhost:6379"
+
+# AES encryption key for mobile numbers (generate with: node -e "console.log(require('crypto').randomBytes(32).toString('hex'))")
+envKey="your_64_char_hex_key_here"
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The envKey key should be in uppercase

(LowercaseKey)